samples/apps/copilot-chat-app/webapi/Options/NotEmptyOrWhitespaceAttribute.cs

```diff
@@ -8,7 +8,7 @@ namespace SemanticKernel.Service.Config;
 /// If the string is set, it must not be empty or whitespace.
 /// </summary>
 [AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
-internal sealed class NotEmptyOrWhitespaceAttribute : ValidationAttribute
+public sealed class NotEmptyOrWhitespaceAttribute : ValidationAttribute
```

Comment:
public for reuse outside the project, right?  Then, we should re-use it outside of the project instead of making a copy!